stdlib/public/stubs/Stubs.cpp

```diff
@@ -172,13 +172,13 @@ static uint64_t swift_floatingPointToString(char *Buffer, size_t BufferLength,
 extern "C" uint64_t swift_float32ToString(char *Buffer, size_t BufferLength,
                                           float Value) {
   return swift_floatingPointToString<float>(Buffer, BufferLength, Value,
-                                            "%0.*g");
+                                            "%0.9g");
```

Comment:
C++ punts to the C standard to define format strings.  The C standard says the following:

> As noted above, a field width, or precision, or both, may be indicated by an asterisk. In this case, an int argument supplies the field width or precision. The arguments specifying field width, or precision, or both, shall appear (in that order) before the argument (if any) to be converted.

So "%0.9g" expects a single value argument, but "%0.*g" expects two arguments, first an integer specifying how many digits to print, and then the value to be printed.